app/src/main/java/com/am/fdamilola/bankingapp/activities/Splash_Screen_Activity.java

```diff
@@ -0,0 +1,53 @@
+package com.am.fdamilola.bankingapp.activities;
+
+import android.content.Intent;
+import android.graphics.Typeface;
+import android.os.Bundle;
+import android.os.Handler;
+import android.widget.TextView;
+
+import com.am.fdamilola.bankingapp.R;
+import com.am.fdamilola.bankingapp.base.classes.TinyGameAppBaseActivity;
+
+public class Splash_Screen_Activity  extends TinyGameAppBaseActivity {
+
+    // Splash screen timer'
+    private TextView mSplashText ;
+    private static int SPLASH_TIME_OUT = 3000;
+
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_splash);
+
+
+        mSplashText = (TextView) findViewById(R.id.splash_text_id);
+        Typeface getfont= Typeface.createFromAsset(getAssets(),
+                "fonts/Candr___.ttf");
+        mSplashText.setTypeface(getfont);
+
+
+
+        new Handler().postDelayed(new Runnable() {
+
```

Comment:
This can introduce a memory leak. Create a Handler variable instead and set to null ondestroy of the activity.